js/src/models/shortcode.js

```diff
@@ -87,6 +87,11 @@ Shortcode = Backbone.Model.extend({
 			content = this.get( 'inner_content_backup' );
 		}
 
+		//Run autop before shooting into view
```

Comment:
Can we have the comment explain _why_ it's doing it? The "what" is more straightforward.